source/ui/Components/Output/Console.qml

```diff
@@ -1,75 +1,73 @@
+            onDoubleClicked: {
+                if (outputComponent.getOutputItem(outputItemIndex)["textMode"] !== 0) {
+                    textarea.clear();
+                }
             }
-
         }
+        function clear() {
+            text = "";
+        }
+    }
 
-        Connections {
-            target: outputComponent
-            // Send when the memory changes (at any address).
-            onMemoryChanged: {
-                var _baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
-                var _mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
-                // Check if the memory address that was changed (at least partly) belongs to
-                // the output item's source space.
-                if ((address+length) >= _baseAddress && _mode === 0) {
+    Connections {
+        target: outputComponent
+        // Send when the memory changes (at any address).
+        onMemoryChanged: {
+            var _baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
+            var _mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
+            // Check if the memory address that was changed (at least partly) belongs to
+            // the output item's source space.
+            if ((address+length) >= _baseAddress && _mode === 0) {
+                it.updateContent(_baseAddress);
+            }else if(_mode !== 0 ){
+                if(address<=_baseAddress && address+length >= _baseAddress){
                     it.updateContent(_baseAddress);
-                }else if(_mode !== 0 ){
-                    if(address<=_baseAddress && address+length >= _baseAddress){
-                        it.updateContent(_baseAddress);
-                    }
                 }
             }
-            // Send when any item's settings where updated.
-            onOutputItemSettingsChanged: {
-                var mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
-                var baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
-                if(mode === oldMode && baseAddress === oldAddress){
-                    return;
-                }
-                it.updateContent(outputComponent.getOutputItem(outputItemIndex)["baseAddress"]);
-                settingsWindowC.updateSettings();
+        }
+        // Send when any item's settings where updated.
+        onOutputItemSettingsChanged: {
+            var mode = outputComponent.getOutputItem(outputItemIndex)["textMode"];
+            var baseAddress = outputComponent.getOutputItem(outputItemIndex)["baseAddress"];
+            if(mode === oldMode && baseAddress === oldAddress){
+                return;
             }
+            it.updateContent(outputComponent.getOutputItem(outputItemIndex)["baseAddress"]);
+            settingsWindowC.updateSettings();
         }
     }
+
     function updateContent(_baseAddress) {
```

Comment:
hmm this was actually just a small fix (~10 lines) about setting the background of the console, everything else was just formatting. So I guess this bug should be in a separate issue (if it really is a bug).